front/app/modules/commercial/flag_inappropriate_content/admin/components/RemoveFlagButton.tsx

```diff
@@ -9,26 +9,30 @@ import messages from './messages';
 
 interface Props {
   processing: boolean;
-  onClick: () => void;
-  activeFlagsCount: number;
+  onRemoveFlags: () => void;
+  selectedActiveFlagsCount: number;
 }
 
-const RemoveFlagButton = ({ processing, onClick, activeFlagsCount }: Props) => {
+const RemoveFlagButton = ({
+  processing,
+  onRemoveFlags,
+  selectedActiveFlagsCount,
+}: Props) => {
   const locale = useLocale();
```

Comment:
Nitpick: if you use Button from `components/UI/Button` you can remove the entire locale logic - import, hook, prop in the button, isNilOrError check. Simplifies the code quite a bit.

Reply:
But we should use one, and in fact components/UI/Button shouldn't actually exist anymore. Once a component is transfered to cl2-component-library, the one in citizenlab should be removed. Because now we have two components to maintain. I logged it in Jira, on the TEC board.

Reply:
I somewhat disagree with this reasoning. The Button in the UI folder is very useful as it simplifies the code in your component significantly (you can remove 4 things!) without requiring a huge amount of mainenace. It's perfectly fine to have wrappers in the app for the components in the component library, when useful. We should keep it and use it, imo.

Reply:
My bad, I didn't look at the component specifically. I thought it was just a copy of the one in cl2-component-library.

Agree with you then. Only, I wouldn't have it in the UI folder anymore, but one level higher in the components folder (since we add data to it and the UI part is in the comp lib. What do you think?